src/components/signon/SignOn.tsx

```diff
@@ -321,6 +324,7 @@ const SignOn = ({ navigation }: SignOnProps) => {
   const emailIsAvailable = useSelector(getEmailIsAvailable)
   const emailIsValid = useSelector(getEmailIsValid)
   const emailStatus = useSelector(getEmailStatus)
+  const isKeyboardVisible = useSelector(getIsKeyboardVisible)
```

Comment:
I think you can use the existing selector for this:
https://github.com/AudiusProject/audius-mobile-client/blob/master/src/store/keyboard/selectors.ts#L4

And then you won't need to make any changes to `useKeyboardListeners`

Reply:
Oh nice! Thanks